lib/prometheus_exporter/middleware.rb

```diff
@@ -54,10 +54,18 @@ def call(env)
   def default_labels(env, result)
     status = (result && result[0]) || -1
     params = env["action_dispatch.request.parameters"]
+    cors = env["rack.cors"]
     action = controller = nil
     if params
       action = params["action"]
       controller = params["controller"]
+    elsif cors && cors.respond_to?(:preflight?) && cors.preflight?
```

Comment:
```suggestion
    elsif (cors = env["rack.cors"]) && cors.respond_to?(:preflight?) && cors.preflight?
```

Minor, but might as well push back the lookup.

```
irb(main):002:0> if (bar=foo[:bar]) && bar == true; puts "hi"; end
hi
```